benchmarks/shipit/Gemfile

```
@@ -0,0 +1,15 @@
+source "https://rubygems.org"
+
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 8.0.2"
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", ">= 2.1"
+# Use the Puma web server [https://github.com/puma/puma]
+gem "puma", ">= 5.0"
+
+# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
+gem "tzinfo-data", platforms: %i[ windows jruby ]
+
+# Reduces boot times through caching; required in config/boot.rb
+gem "bootsnap", require: false
+gem "shipit-engine", path: "/Users/byroot/src/github.com/Shopify/shipit-engine"
```

```suggestion
gem "shipit-engine"
```

The CI seems to be failing on this.

---

🤦 I meant to cut a new release. I'll do it now.